hs_modflow_modelinstance/migrations/0001_initial.py

```diff
@@ -0,0 +1,194 @@
+# -*- coding: utf-8 -*-
```

Comment:
@pkdash I tried to create just only one migration file for this new resource type but every time I do that I get this error: 

django.db.migrations.state.InvalidBasesError: Cannot resolve bases for [<ModelState: 'hs_modflow_modelinstance.ModelOutput'>, <ModelState: 'hs_modflow_modelinstance.ExecutedBy'>, <ModelState: 'hs_modflow_molow_modelinstance.MODFLOWModelInstanceMetaData'>]

That is because this resource type is inherited from the Model Instance resource type. To solve this error, I have to do the migration in two steps ( @pkdash  we discussed that before). I generated two new migration files.